### PR TITLE
Linter fixes | remove unneeded ignore comments

### PR DESCRIPTION
**Pull request by anselmbradford**
We have an error in our linter. This PR adds an ignore comment for that.

As our eslint config has evolved, some ignore comments can be removed. 

## Changes

- Add eslint ignore comment for redefine of `global`.
- Auto fix linter issues
- Remove unneeded ignore comments
- Patch update eslint and eslint-plugin-jsdoc

## How to test this PR

1. `yarn lint` should have 146 warnings and no errors.

## Screenshots

before:
Screen Shot 2023-03-14 at 3 26 08 PM

after:
Screen Shot 2023-03-14 at 3 23 07 PM

### CODE REVIEW

File: cfgov/unprocessed/js/organisms/MortgagePerformanceTrends/utils.js [hunk 1]
```diff
@@ -378,7 +378,7 @@ const utils = {
     console.groupCollapsed('previous state:');
     console.log(JSON.stringify(store.getState(), '', '\t'));
     console.groupEnd();
-    var result = next(action); // eslint-disable-line
+    const result = next(action);
```
Comment (anselmbradford) [inline cfgov/unprocessed/js/organisms/MortgagePerformanceTrends/utils.js:381]: @contolini any problem with this change?

**Comment by anselmbradford** [comment]
Coverage failure is N/A. It is because I touched the files.

File: cfgov/unprocessed/js/organisms/MortgagePerformanceTrends/utils.js (hunk 1 above)
Comment (contolini) [reply cfgov/unprocessed/js/organisms/MortgagePerformanceTrends/utils.js:381]: 👍

**Review by contolini — APPROVED** [review]
/* eslint-disable everything */

**Pull request merged**